lib/Munin/Master/HTML.pm

```diff
@@ -583,6 +583,7 @@ RENDERING:
 		use JSON;
 		print encode_json( \%template_params );
 	}
+	$dbh->disconnect();
```

Comment:
This isn't needed, as it will be done automatically when `$dbh` gets out of scope.

And, if you want to be explicit, you forgot an occurence above (when replying 404). So better leave it to RAII.

Reply:
Indeed: I was not aware of the implicit resource cleanup.
Should I remove all the "disconnect" statements again in order to not pollute the exemplary character of the code?

Reply:
Yes, please 👍